pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -530,6 +545,11 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 					Resources: sets.NewString("images"),
 				},
 				{
+					Verbs:     sets.NewString("update"),
```

Comment:
The image object is "immutable". Its signatures array is an exception. It's similar to `imagestream/status`. You get the signatures when you `GET image` the same way you get the IS status on `GET imagestream`.

Initially we considered on API route in the registry for pushing signatures. However, that won't be the case in the near future. I'll remove this entry from this PR and re-add once needed.